offset_manager_test.go

```diff
@@ -0,0 +1,305 @@
+package sarama
+
+import (
+	"testing"
+	"time"
+)
+
+var (
+	broker, coordinator *mockBroker
+	testClient          Client
+	config              *Config
+)
```

Comment:
The only reason I can remember that I went this route was to simplify return values for those init functions. Shouldn't be too difficult to fix...